spec/mailers/user_mailer_spec.rb

```diff
@@ -40,17 +40,23 @@
 
   describe 'reservation_status_update' do
     it 'sends to the reserver' do
+      # make it a request so the email sends
+      @res.update_attributes(flags: Reservation::FLAGS[:request])
```

Comment:
I think this is ok here, although the `reservation_status_update` method is getting a bit too wonky for me; I'll comment more above.